src/wp-content/themes/twentynineteen/style-editor.css

```diff
@@ -1604,4 +1604,4 @@ ul.wp-block-archives li ul,
   padding-left: 0;
   padding-right: 0;
   width: 100%;
-}
+}
```

Comment:
Please restore the empty line at the end of the file.

(Whoever commits the change could compile this stylesheet from the SCSS, but the file should remain unchanged if it is not compiled within the pull request.)

Reply:
I have Restored Last empty line.